dts/bindings/display/sitronix,st7735r.yaml

```diff
@@ -115,3 +115,9 @@ properties:
       type: int
       default: 0x07
       description: Display Inversion Control
+
+    swap-color-order:
```

Comment:
If this is trying to fix an LCD issue (it works in RGB when instructed to do BGR and viceversa), the name of this property is misleading. Maybe `rgb-is-inverted`, and indicate clearly that this can be used for buggy displays.

Reply:
Thanks for the proposal. Apply it and update the description.